app/src/main/res/values/arrays.xml

```diff
@@ -470,4 +470,31 @@
         <item>vn</item>
     </string-array>
 
+    <string-array name="ymin_entries">
+        <item>40 mg/dL (2.2 mmol/L) ></item>
+        <item>50 mg/dL (2.8 mmol/L)</item>
+        <item>60 mg/dL (3.3 mmol/L)</item>
+        <item>70 mg/dL (3.9 mmol/L)</item>
+    </string-array>
+    <string-array name="ymin_values">
+        <item>40</item>
+        <item>50</item>
+        <item>60</item>
+        <item>70</item>
+    </string-array>
+    <string-array name="ymax_entries">
+        <item>210 mg/dL (11.7 mmol/L)</item>
+        <item>230 mg/dL (12.8 mmol/L)</item>
+        <item>250 mg/dL (13.9 mmol/L) ></item>
```

Comment:
Is the " >" intended? (there's another one above btw)

Reply:
Yes, I would like us all to use a symbol for marking the default setting.
Have you ever looked at a setting you had changed and wasn't sure what it was before you ever changed it?

I asked if we already had an agreement.  I never got a response.  So, I used that.  I think we should agree on some symbol and all use it.
We can use def or an acronym.  But, the problem is I think we should minimize the size of all titles because sometimes titles don't wrap.  That's why I hope we can agree on a single symbol representing the default.